### ISSUE TITLE
downloading subtitles even when they are embedded

### ISSUE THREAD

**Issue by vicks619** [SickChill/SickChill#6300]
2020-04-23 12:26:27 INFO     FINDSUBTITLES :: Downloaded und subtitles for Ghostwriter (2019) S01E01
2020-04-23 12:26:27 INFO     FINDSUBTITLES :: Searching for missing subtitles of Ghostwriter (2019) S01E02
2020-04-23 12:26:39 INFO     FINDSUBTITLES :: Downloaded und subtitles for Ghostwriter (2019) S01E02
2020-04-23 12:26:39 INFO     FINDSUBTITLES :: Searching for missing subtitles of Ghostwriter (2019) S01E03
2020-04-23 12:26:49 INFO     FINDSUBTITLES :: Downloaded und subtitles for Ghostwriter (2019) S01E03
2020-04-23 12:26:49 INFO     FINDSUBTITLES :: Searching for missing subtitles of Ghostwriter (2019) S01E04
2020-04-23 12:27:01 INFO     FINDSUBTITLES :: Downloaded und subtitles for Ghostwriter (2019) S01E04
2020-04-23 12:27:01 INFO     FINDSUBTITLES :: Searching for missing subtitles of Ghostwriter (2019) S01E05
2020-04-23 12:27:11 INFO     FINDSUBTITLES :: Downloaded und subtitles for Ghostwriter (2019) S01E05
2020-04-23 12:27:11 INFO     FINDSUBTITLES :: Searching for missing subtitles of Ghostwriter (2019) S01E06
2020-04-23 12:27:21 INFO     FINDSUBTITLES :: Downloaded und subtitles for Ghostwriter (2019) S01E06
2020-04-23 12:27:21 INFO     FINDSUBTITLES :: Searching for missing subtitles of Ghostwriter (2019) S01E07
2020-04-23 12:27:30 INFO     FINDSUBTITLES :: Downloaded und subtitles for Ghostwriter (2019) S01E07

@miigotu whatever you are saying makes sense, but that is not the case here. you can see only english selected as language. and now it has finally managed to download the subtitles
_Originally posted by @vicks619 in https://github.com/SickChill/SickChill/issues/6299#issuecomment-618218012_

**Comment by vicks619** [comment]
at the same time it has downloaded for prison break s01, episode 10, 11, 12, 13,17,18,19,20,21,22 again but failed to even look for episode 1,2,4,5,6,7 even once. Also, here it has not looked for episode 3,14,15,16 as they have embedded subtitles which is correct but not was the case of ghostwriters
![image](https://user-images.githubusercontent.com/63958559/80070900-5efa3800-8561-11ea-9222-45ea53ed6a03.png)
![image](https://user-images.githubusercontent.com/63958559/80071002-86510500-8561-11ea-9dad-8a9c5c924e15.png)
![image](https://user-images.githubusercontent.com/63958559/80071051-9e288900-8561-11ea-8941-d08cd9e6ccb9.png)

**Comment by vicks619** [comment]
ghostwriters episode 2-6 has all possible language subtitles embedded and episode 1 has at least English which is the only language selected for subtitle downloads in subtitle settings.
![image](https://user-images.githubusercontent.com/63958559/80071873-ff9d2780-8562-11ea-9f2e-26ecc67f2e16.png)
![image](https://camo.githubusercontent.com/1fb04a62b116b853579c6548d38138e11ed6cbbd/68747470733a2f2f692e696d6775722e636f6d2f3771414c62327a2e706e67)

**Comment by miigotu** [comment]
WHY DO YOU KEEP OPENING ISSUES. Just because an issue is closed does not mean you can't comment in it. Please dont make me block you.

**Comment by vicks619** [comment]
@miigotu because first issue was for searching only, but now it is downloading as well how can it close when the issue persists. If you don't want people to report issue, I am fine with it but don't say issue is closed and resolved. I appreciate your hard work but closing issues without understanding is dictatorship and block me if you want, I am not coming back here to report anything if this is the attitude.  why would I take extra efforts to post screenshots and details and waste my time when you are not even reading it and threatening to block me. My intention was not to create unnecessary tracks but report the bug. Thanks for your continuous efforts on fixing bugs

**Comment by miigotu** [comment]
If you open 50 issues a day, it is not helpful. It is a dictatorship, unless you want to contribute some code. Meaning, I am the only one doing the work, and you are not my supervisor.
I have people with actual broken shit and you nitpicking is taking my very limited time away from real problems. You are not the only person in the world.

**Comment by miigotu** [comment]
And btw, the `und` it is talking about is just the found embedded subtitle, that has a language tag it cannot understand. Subtitle support has been the same, and working, for many years.

**Comment by vicks619** [comment]
> If you open 50 issues a day, it is not helpful. It is a dictatorship, unless you want to contribute some code. Meaning, I am the only one doing the work, and you are not my supervisor.
> I have people with actual broken shit and you nitpicking is taking my very limited time away from real problems. You are not the only person in the world.

Neither I have opened 50 issues in a day nor I was expecting a quick fix, but you closed it anyway. I opened 3 issues, out of which I closed one myself and 2 were force closed by you. You are correct I am not your supervisor. I am not the only person in the  world neither I pay you to fix the issues and not even a programmer who can fix things myself. I reported incident which I felt is a problem but will not report now onward.

Regarding the `und` explanation, you have given, if it is embedded subtitle then why there is a .srt file created in the folder after it says searching & downloaded und subtitle. And why it skip to search for subtitles for specific episodes of prison break. I understand subtitle support is working for many years but not it is not bug free. 

I am sorry if you feel offended/irritated. Wish you all the best
